### ISSUE TITLE
i want to a real example, but not  validating a mdoel 

### ISSUE THREAD

**Issue by dlml** [Wheest/flatbuffer-mobilenet#1]

**Comment by Wheest** [comment]
Hi there.  This is something I'm still trying to figure out.

Being able to read `.tflite` flatbuffer data is a planned feature for a project I'm working on, but it's on the back burner for now.

I guess the features that I'd want would being able to get layer hyper-parameters as primitive variables, and weights in some format, such as an `std::vector` for each layer. 

What is it that you're trying to do?  Perhaps we can try and work this out together.

There's a repo [https://github.com/SF-Zhou/tflite-model-parse](here), which might have at least what I'm looking for.

**Comment by dlml** [comment]
i have found this example https://github.com/SF-Zhou/tflite-model-parse，but this example only can validate tflite model，but i want use c++ to input a real image and return a real label

**Comment by Wheest** [comment]
Sure.  Have you got any thoughts on what will be doing the tensor operations?

The `.tflite` flatbuf should contain the weights and hyperparamters for the network, but are you going to be implementing say the depthwise convolutional layer yourself in C++, use a library implementation, or get the full inference from an external framework?
